lib/elixir/lib/module.ex

```diff
@@ -345,8 +345,9 @@ defmodule Module do
   ### Custom attributes
 
   In addition to the built-in attributes outlined above, custom attributes may
-  also be added. A custom attribute is any valid identifier prefixed with an
-  `@` and followed by a valid Elixir value:
+  also be added. Custom attributes are expressed using the `@` operator followed
```

Comment:
```suggestion
  also be added. Custom attributes are expressed using the `@/1` operator followed
```

This should link, right?